### ISSUE TITLE
Training from scratch 

### ISSUE THREAD

**Issue by shub2k** [Guillem96/efficientdet-tf#33]
hey @Guillem96 Thanks for the work i was training the model from scratch only the effnet backbone has the imagenet pretrained weights , i trained the model for 30 epochs on a custom dataset with 12 classes but was getting very low confidence scores and bad predictions , can you share your training configs and share the results ,

**Comment by thomas-chia** [comment]
Hey @mu745511 I am also having the same issue, were you able to figure it out? I am getting around 0.7 for my total loss on PascalVOC. @Guillem96 any help would be much appreciated!

**Comment by shub2k** [comment]
hi @IdeaKing i believe it requires 250-300 epochs to get some decent predictions even though model was overfitting as train and validation metric or loss was not correlating i got 0.59 map@45 and score threshold 0.2 on training set and 0.36 map with same metric config on voc dataset , i believe first training from scratch on coco and then using the coco trained weights to further train voc or other custom dataset will boost the model performance a lot .....
